### PR TITLE
Amp article style tweaks

### PR DESCRIPTION
**Pull request by Pearson82**
Tweaks to byline area, caption and rich link

### CODE REVIEW

**Comment by NataliaLKB** [comment]
@Pearson82 could you provide a screen shot?

**Comment by Pearson82** [comment]
Blockquote spacing...

Before
screen shot 2015-09-24 at 10 11 27

After
screen shot 2015-09-24 at 10 11 43

Caption and byline area...

Before
screen shot 2015-09-24 at 10 09 19

After
screen shot 2015-09-24 at 10 09 35


Rich links, Mobile + Phablet breakpoints
Before
screen shot 2015-09-24 at 10 12 59

screen shot 2015-09-24 at 10 14 15

After

screen shot 2015-09-24 at 10 13 13
screen shot 2015-09-24 at 10 13 35


Also increased margins on page from 10px to 20px above 480px breakpoint.

**Comment by johnduffell** [comment]
brilliant :+1: thanks Chris!

File: common/app/views/fragments/amp/stylesheets/richLinks.scala.html [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+}
+
+@@media (max-width: 29.99rem) {
+    .element-rich-link--not-upgraded {
+    width: 8.125rem;
+    font-size: 0.875rem;
+    }
+}
+
+@@media (min-width: 30rem) {
```
Comment (johnduffell) [inline common/app/views/fragments/amp/stylesheets/richLinks.scala.html]: how about 29.995rem? :stuck_out_tongue:

**Pull request merged**